smartsim/settings/arguments/launch/lsf.py

```diff
@@ -148,5 +148,5 @@ def set(self, key: str, value: str | None) -> None:
             )
             return
         if key in self._launch_args and key != self._launch_args[key]:
```

Comment:
Another "out of scope" sanity check
```suggestion
        if key in self._launch_args and value != self._launch_args[key]:
```